fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -236,6 +254,8 @@ protected void setJobProperties(Map<String, String> properties) throws DdlExcept
         }
     }
 
+    abstract void setAuthorizationInfo() throws MetaNotFoundException;
```

Comment:
It's strange that a set function without any input param. I think that this can be done in class's constructor?